website/widgets/form/table.md

```diff
@@ -102,6 +102,29 @@ q.page['example'] = ui.form_card(box='1 1 3 4', items=[
 ])
 ```
 
+### Select event
+
+Sometimes it can be useful to update other components immediately based on the current row selection (e.g. enable/disable the submit button).
+This is where the `select` event comes into the game. When the `select` event is registered, it is emitted every time when there is a change
+in the current table row selection. It can be accessed through `q.events.table_name.select` and contains the list of all selected item names, where `table_name`
```

Comment:
```suggestion
in the current table row selection. It can be accessed through `q.events.<table_name>.select` and contains the list of all selected item names, where `table_name`
```